check.py

```
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+import os
+import openai
```

Ruff (0.8.2)

`openai` imported but unused

Remove unused import: `openai`

(F401)